src/server.js

```diff
@@ -25,19 +23,4 @@ const server = Server({
   db: dbWithCaching,
 });
 
-server.run(process.env.PORT || 8000, () => {
```

Comment:
Errors raised while running on App Engine will automatically be forwarded to GCE Error Reporting

Reply:
nice 👍